etcdmain/help.go

```diff
@@ -66,6 +66,8 @@ member flags:
 		comma-separated whitelist of origins for CORS (cross-origin resource sharing).
 	--quota-backend-bytes '0'
 		raise alarms when backend size exceeds the given quota (0 defaults to low space quota).
+	--max-txn-ops '128' 
+		maximum operations per txn that etcd server allows; defaults to 128.
```

Comment:
the default is already given in the line above, don't restate
`maximum number of operations  permitted in a transaction`